app/src/main/java/com/tistory/dividendcalendar/presentation/stock/StockViewModel.kt

```diff
@@ -1,49 +1,31 @@
 package com.tistory.dividendcalendar.presentation.stock
 
+import androidx.hilt.lifecycle.ViewModelInject
 import androidx.lifecycle.MediatorLiveData
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.asLiveData
-import com.tistory.dividendcalendar.DividendCalendarApplication
-import com.tistory.dividendcalendar.R
-import com.tistory.domain.model.StockWithDividendItem
 import com.tistory.domain.repository.StockWithDividendRepository
-import kotlinx.coroutines.flow.debounce
 
-class StockViewModel(
-    private val stockWithDividendRepository: StockWithDividendRepository
+class StockViewModel @ViewModelInject constructor(
+    stockWithDividendRepository: StockWithDividendRepository
 ) : ViewModel() {
 
-    val stockItems = stockWithDividendRepository.getStockItems().debounce(500).asLiveData()
-
-    val topTitleLiveData = MediatorLiveData<String>().apply {
-        addSource(stockItems) {
-            val title = if (it.isEmpty()) {
-                DividendCalendarApplication.INSTANCE.getString(R.string.app_name)
-            } else {
-                getDividendMonthlyTitle(it)
-            }
-
-            postValue(title)
-        }
-    }
+    //TODO using debounce make error in test
+    //java.lang.AbstractMethodError: kotlinx.coroutines.test.internal.TestMainDispatcher.invokeOnTimeout
+    //val stockItems = stockWithDividendRepository.getStockItems().debounce(500).asLiveData()
+    val stockItems = stockWithDividendRepository.getStockItems().asLiveData()
 
     val isVisibleEmptyViewLiveData = MediatorLiveData<Boolean>().apply {
         addSource(stockItems) {
             postValue(it.isEmpty())
         }
     }
 
-    private fun getDividendMonthlyTitle(items: List<StockWithDividendItem>): String {
-        val amount = getDividendMonthly(items)
-        return String.format(
-            DividendCalendarApplication.INSTANCE.getString(R.string.total_dividend_monthly), amount
-        )
-    }
-
-    private fun getDividendMonthly(items: List<StockWithDividendItem>): Float {
+    fun getDividendMonthly(): Float {
         var totalAmount = 0f
 
-        items.forEach { item ->
+        val items = stockItems.value
+        items?.forEach { item ->
```

Comment:
```suggestion
        stockItems?.value.forEach { item ->
```
코드 스타일의 차이일 것 같지만 ,,  변수선언 없이 forEach 사용하는 것은 어떨지 제안드려봅니다 😊

Reply:
반영하겠습니다~!

Reply:
d3c9ff1